src/components/types/Route.ts

```diff
@@ -36,3 +39,9 @@ export type RouteCalendarActive = {
 };
 
 export type RouteTab = 'calendar' | 'list' | 'map' | 'app';
```

Comment:
Use inside `RouteTab` type enum, please. Make it in the new PR, please.

Reply:
Updated in PR #515.